api/namex/resources/name_requests/resource.py

```diff
@@ -294,12 +285,15 @@ def update_request_in_nro(self, name_request, on_success=None):
                 })
 
             return self.on_nro_update_complete(name_request, on_success, nro_warnings)
-        # Handle any changes where ONLY state is changed
-        elif name_request.stateCd in [State.CANCELLED]:
-            # TODO: It might be a good idea to set an env var for this... as NRO can't run in local tests
-            nro_warnings = self.nro_service.change_nr(name_request, {
-                NROChangeFlags.STATE.value: True
-            })
+        elif name_request.stateCd in [State.COND_RESERVE, State.RESERVED, State.CONDITIONAL, State.APPROVED, State.CANCELLED]:
+            if current_app.config.get('DISABLE_NAMEREQUEST_NRO_UPDATES', 0) == 1:
+                # Ignore update to NRO if NRO updates [DISABLE_NAMEREQUEST_NRO_UPDATES] are explicitly disabled in your .env
+                nro_warnings = None
+            else:
+                nro_warnings = self.nro_service.change_nr(name_request, {
+                    NROChangeFlags.REQUEST.value: True,
+                    NROChangeFlags.STATE.value: True
```

Comment:
We only update the state in NRO when the state=CANCELLED. Not for APPROVED or CONDITIONAL. 
elif name_request.stateCd in [State.CONDITIONAL, State.APPROVED, State.CANCELLED]:

....
if State.State.CANCELLED: NROChangeFlag,STATE.value = True 
else: NROChangeFlags.REQUEST.value = True

nro_warnings = self.elf.nro_service.change_nr(name_request, {
                    NROChangeFlags}